eng/docker-compose/kafka-opensearch.yml

```diff
@@ -117,3 +127,7 @@ volumes:
   kafka-opensearch-sink-logs:
   kafka-opensearch-sink-config:
   opensearch-data1:
+
+networks:
+  dms:
+    external: true
```

Comment:
I used this external network so that keycloak can be started on its own before (e.g. while manually running the applications from VS / VS Code), and the service will still be accessible when starting DMS and Config Service using their compose files.